RetailCoder.VBE/Navigation/CodeExplorer/CodeExplorerComponentViewModel.cs

```diff
@@ -47,7 +47,53 @@ public CodeExplorerComponentViewModel(CodeExplorerItemViewModel parent, Declarat
                                 .OrderBy(item => item.QualifiedSelection.Selection.StartLine)
                                 .Select(item => new CodeExplorerMemberViewModel(this, item, grouping)))
                                 .ToList<CodeExplorerItemViewModel>();
-            
+
+            _name = _declaration.IdentifierName;
+
+            var component = declaration.QualifiedName.QualifiedModuleName.Component;
+            if (component.Type == vbext_ComponentType.vbext_ct_Document)
```

Comment:
is there the slightest chance that this could result in document nodes looking like **IdentifierName ()**?